package.json

```diff
@@ -15,32 +15,40 @@
     "@material-ui/core": "^4.11.0",
     "@material-ui/lab": "^4.0.0-alpha.56",
     "@react-firebase/auth": "^0.2.10",
-    "@react-spectrum/combobox": "^3.0.0-alpha.2",
+    "@react-spectrum/color": "^3.0.0-beta.1",
+    "@react-spectrum/combobox": "^3.0.0-beta.0",
+    "@react-spectrum/slider": "^3.0.0",
+    "@react-spectrum/table": "^3.0.0-alpha.9",
     "@react-spectrum/tabs": "^3.0.0-alpha.3",
     "@spectrum-css/typography": "^3.0.0-beta.0",
     "@spectrum-icons/illustrations": "^3.2.1",
     "@spectrum-icons/workflow": "^3.2.0",
+    "@spectrum-web-components/accordion": "^0.1.4",
     "availity-reactstrap-validation": "2.6.1",
     "axios": "0.19.2",
     "bootstrap": "4.4.1",
     "firebase": "^8.2.0",
+    "jquery": "^3.5.1",
```

Comment:
remove jquery

Reply:
add a comment "jquery will be removed in phase 2"

Reply:
done